cmd/vela-cli/add.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"github.com/go-vela/cli/action"
+
+	"github.com/urfave/cli/v2"
+)
+
+// addCmds defines the commands for creating resources.
+var addCmds = &cli.Command{
```

Comment:
do we want to annotate these to avoid the linter going nuts? like add a
```go
// nolint:deadcode // this command is used in cmds.go
```
or something like it? seems like the linter is not smart enough to know it's being used in `cmds.go`?

Reply:
@wass3r so the linter is accurate when it states these commands aren't being used currently.

All these commands being added aren't directly referenced in a codepath because I'm trying to keep this PR smaller.

Right now, our `main.go` file is referencing the `cmds.go` file you are talking about, but that file is located outside the `cmd/vela-cli` directory. These commands being added are directly added to the `cmd/vela-cli` directory 👍

Reply:
Ah, my fault. I must have thought it was all in the same folder. I was just trying to avoid seeing this pop up in future PRs in the “files that didn’t change” section. Sounds like there’s more changes coming :+1: 

Thanks